src/pages/page-2.js

```diff
@@ -1,16 +1,89 @@
+                            />
+                            <FunctionDescription
+                              period="2010 - heden"
+                              job="Front-End Developer @ Coop Supermarkten"
+                              description="Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus."
+                            />
+                        <TitleComponent title="Opleidingen" />
+                            <FunctionDescription
+                              period="2017 - 2018"
+                              job="Strategic Entrepreneurship @ Erasmus Universiteit Rotterdam"
+                              description="Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus."
+                            />
+                            <FunctionDescription
+                              period="2011 - 2016"
+                              job="Bedrijfskunde @ Erasmus Universiteit Rotterdam"
+                              description="Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus."
+                            />
+                            <FunctionDescription
+                              period="2011 - 2016"
+                              job="Bedrijfskunde @ Erasmus Universiteit Rotterdam"
+                              description="Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus."
+                            />
+                    </div>
+                    <div className="column column--orange">
+                        <TitleComponent title="Info" />
+                            <UserInfo item="email" value="sander.vanrijsoort@incentro.com" />
+                            <UserInfo item="telefoon" value="06-43499341" />
+                            <UserInfo item="geboortedatum" value="23 april 1993" />
+
+                            {/* Lijsten kunnen gemaakt worden met vierkantjes, cijfers of niks */}
+                            <List list={kwaliteiten} title="Kwaliteiten" />
+                            <List list={skills} title="Skills" type="decimal" />
+                            <List list={optioneel} title="Optioneel" type="none" />
+
+                            <img src= {logoWhite} alt="logo_white" style={{ height: "20px",
```

Comment:
Is het hier nodig om inline style te gebruiken? het liefst deze waardes vanuit een class laten komen.